diff --git a/dist/index.js b/dist/index.js
index 5ebd3756860e75d89ef13b683310a4fe066bdd62..287b23d26e3995f8e51c3b7f935e04a66fbe25e5 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -3657,7 +3657,8 @@ var OpenRouterChatLanguageModel = class {
             function: {
               name: tool2.name,
               description: tool2.description,
-              parameters: tool2.inputSchema
+              parameters: tool2.inputSchema,
+              strict: tool2.strict
             }
           }, eagerInputStreaming != null && {
             eager_input_streaming: eagerInputStreaming
diff --git a/dist/internal/index.js b/dist/internal/index.js
index ba7bca933a29fd4729b5e9f525512dbed39e6e6f..87bdfbdadc78391e3c38f2dcf99f1bcfdbedaf94 100644
--- a/dist/internal/index.js
+++ b/dist/internal/index.js
@@ -3581,7 +3581,8 @@ var OpenRouterChatLanguageModel = class {
             function: {
               name: tool.name,
               description: tool.description,
-              parameters: tool.inputSchema
+              parameters: tool.inputSchema,
+              strict: tool.strict
             }
           }, eagerInputStreaming != null && {
             eager_input_streaming: eagerInputStreaming
